### PR TITLE
Docs/beta

### PR DESCRIPTION
**Pull request by jorgeucano**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe:

## What is the current behavior?

Change styling and fix some docs

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
Now looks really good 

darkmode

### CODE REVIEW

File: docs/scully-cmd-line.md [hunk 1]
```diff
@@ -33,13 +34,13 @@ npx scully serve
...
-## noWatch
+## Watch
 
 ```bash
-npx scully --nw
+npx scully --watch
 ```
 
-By default, Scully has the watchMode active. Use this flag to run scully only once.
+By default, Scully has the watchMode in false. You need add this flag for use Watch Mode.
```
Comment (geromegrignon) [inline docs/scully-cmd-line.md:43]: I would suggest to change 
from : `You need add this flag for use Watch Mode.`
to : `You need to add this flag to use Watch Mode.`

File: docs/scully-cmd-line.md [hunk 2]
```diff
@@ -133,6 +134,20 @@ npx scully serve/watch --ssl --ssl-key=./url/to/file
 
 Adds a url to an ssl key file for a server with SSL.
 
+## highlight
+
+```bash
+npx scully serve/watch --hl
+```
+
+Add highlight.js render into the markdown's files.
```
Comment (geromegrignon) [inline docs/scully-cmd-line.md:143]: Change
from : `Add highlight.js render into the markdown's files.`
to : `Add highlight.js to render into the markdown's files.`

File: docs/scully-cmd-line.md [hunk 3]
```diff
@@ -133,6 +134,20 @@ npx scully serve/watch --ssl --ssl-key=./url/to/file
...
+
+If you use this flag, you need add the css into the index.html:
```
Comment (geromegrignon) [inline docs/scully-cmd-line.md:145]: Change
from : `If you use this flag, you need add the css into the index.html:`
to : `If you use this flag, you need to add the css into the index.html:`

File: docs/scully-configuration.md [hunk 4]
```diff
@@ -32,22 +32,48 @@ The `scully.config.js` file's structure is shown below:
...
 ```ts
 export interface ScullyConfig {
-  projectRoot: string;
+  /** the name of the project we are using. Provided by Scully itself */
+  projectName: string;
+  /** the folder where project is. Can be any off the projects in a repo, read from angular.json */
+  projectRoot?: string;
+  /** the folder where the project sources resides, read from angular.json */
+  sourceRoot?: string;
+  /** Array with string ID's of the content-renderes that will be run on all routes */
```
Comment (geromegrignon) [inline docs/scully-configuration.md:45]: Change
from : `content-renderes`
to : `content-renderers`

File: docs/scully-configuration.md [hunk 5]
```diff
@@ -32,22 +32,48 @@ The `scully.config.js` file's structure is shown below:
...
+  defaultPostRenderers: string[];
+  /** the root of the project (where angular.json lives) */
   homeFolder: string;
+  /** the destination off the Scully generated files */
```
Comment (geromegrignon) [inline docs/scully-configuration.md:49]: Change
from : `the destination off`
to : `the destination of`

File: docs/scully-configuration.md [hunk 6]
```diff
@@ -32,22 +32,48 @@ The `scully.config.js` file's structure is shown below:
...
   outDir?: string;
+  /** the place where distribution files of the project are. Should be a subfolder of dist. */
   distFolder?: string;
+  /** transferState only inlined into page, and not written into separate data.json */
+  inlineStateOnly: boolean;
+  /** routes that needs additional processing have their configuration in here */
```
Comment (geromegrignon) [inline docs/scully-configuration.md:55]: Change
from : `routes that needs`
to : `routes that need`

**Comment by jorgeucano** [comment]
thanks @geromegrignon i will create another pr with this changes!

**Pull request merged**